packages/cli/package.json

```diff
@@ -57,8 +57,8 @@
     "uuid": "3.3.2"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "^1.23",
     "@oclif/test": "^1.2.4",
+    "@rasphilco/dev-cli": "^0.1.0",
```

Comment:
@jheikes515 this is a big blocker for this work. We don't know what changes were made to `dev-cli` here, and we definitely don't want to be using a dependency that's been pushed up to a personal npm account.